builders/ethereum/precompiles/features/staking.md

```diff
@@ -459,21 +459,6 @@ The Solidity interface includes the following functions:
 
         None.
 
-As of runtime 2400, the following methods are **deprecated**:
-
-??? function "**delegate**(*address* candidate, *uint256* amount, *uint256* candidateDelegationCount, *uint256* delegatorDelegationCount) - makes a delegation in support of a collator candidate and automatically sets the percent of rewards to auto-compound to `0`. Use `delegateWithAutoCompound` instead"
-
-    === "Parameters"
-
-        - `candidate` - address of the candidate to delegate to
-        - `amount` - uint256 amount to delegate
-        - `candidateDelegationCount` - uint256 current number of delegations for the candidate
-        - `delegatorDelegationCount` - uint256 current number of delegations from the delegator
-
-    === "Returns"
-
-        None.
-
 As of runtime 1800, the following methods are **deprecated** and, as of runtime 2500, have been removed:
 
 ??? function "**scheduleLeaveDelegators**() - schedules a request to leave the set of delegators and revoke all ongoing delegations. Scheduling the request does not automatically execute it. There is an [exit delay](#exit-delays) that must be waited before you can execute the request via the `executeLeaveDelegators` extrinsic. Use the [batch utility](/builders/ethereum/precompiles/ux/batch/){target=\_blank} with `scheduleRevokeDelegation` for all delegations instead"
```

Comment:
These list the removed functions. So, we should follow suit and either remove all the removed ones or restore the deleted section and mark it as removed